dev/bs-config.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
         "cwd": "../"
     },
     "files": [
+        "static/src/javascripts/*.js",
         "static/src/javascripts/**/*.js",
```

Comment:
you'd think so! but it wasn't picking up top level (`core.js` etc) file changes without it...